src/platforms/react-native/performance/instrumentation/user-interaction-instrumentation.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: User Interaction Instrumentation
+sidebar_order: 20
+sdk: sentry.javascript.react-native
+
+description: "Learn what kind of user interaction can be captured"
+---
+
+The UI instrumentation, once enabled, captures transactions and adds breadcrumbs for touch interactions. Gesture support using React Native Gesture Handler is available using the `traceGesture` wrapper. 
+
+## Touch Events
+
+The SDK composes the transaction name from the name of a displayed screen (for example `LoginScreen`) and a label of the element user interacted with (for example `login_button`).
+
+If the element doesn't have `sentry-label` or a user-defined label, the transaction won't be captured.
+
+<Note>
+Capturing UI transactions requires that you first set up <Link to="/platforms/react-native/performance/instrumentation/automatic-instrumentation/#enable-routing-instrumentation">routing instrumentation</Link> if you haven't already.
+</Note>
+
+The UI instrumentation tracing is disabled by default, but you can enable it by setting the `enableUserInteractionTracing` options to `true` and wrapping your root component.
+
+```js
+import * as Sentry from '@sentry/react-native'
+
+Sentry.init({
+  // ...
+  integrations: [
+    new Sentry.ReactNativeTracing({
+      enableUserInteractionTracing: true,
+      routingInstrumentation,
+      // ...
+    }),
+  ],
+})
+
+const App = () => <View>Your App</View>
+
+export default Sentry.wrap(App)
+```
+
+The label by which UI elements are identified is set by `labelName` option in `Sentry.wrap`. If no value is supplied `sentry-label` is used. If an element can't be identified the transaction won't be captured.
+
+```js
+export default Sentry.wrap(App, { touchEventBoundaryProps: { labelName: 'my-label' } });
+```
+
+<Note>
+If the UI transaction has idled, but didn't have any child spans added, it will be dropped.
+</Note>
+
+Transactions are always bound to the Scope automatically if there's none set. Because of that, you can create spans using [manual instrumentation](/platforms/react-native/performance/instrumentation/custom-instrumentation/#retrieve-a-transaction), and those spans will be automatically associated with the running UI transaction. The `idleTimeoout` defaults to `1000` milliseconds (one second).
+
```

Comment:
```suggestion
Because transactions are automatically bound to scope if no spans are set, you can create spans using [manual instrumentation](/platforms/react-native/performance/instrumentation/custom-instrumentation/#retrieve-a-transaction), which will automatically associate those spans with the running UI transaction. Note that the `idleTimeoout` for running UI transactions defaults to `1000` milliseconds (one second).

```
This section is a bit difficult for me to understand. Are transactions bound automatically to scope if no spans are set or are they bound automatically if no scope is set? What happens if it is set? Also, not making a clear connection as to why that means that you can create spans using manual instrumentation. It could just be that I am unfamiliar with the concept, but just wanted to bring it up in case you feel we could add some additional context.

Reply:
Thanks for pointing it out. I've simplified it, what do you think? Is it okay like this? @lizokm 

```suggestion
Because transactions are automatically bound to scope, you can create child spans using [manual instrumentation](/platforms/react-native/performance/instrumentation/custom-instrumentation/#retrieve-a-transaction). Note that the `idleTimeoout` for running UI transactions defaults to `1000` milliseconds (one second).
```